.buildkite/scripts/steps/scalability/benchmarking.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+#.buildkite/scripts/bootstrap.sh
+echo "--- yarn kbn reset && yarn kbn bootstrap"
+yarn kbn reset && yarn kbn bootstrap
+
+GCS_ARTIFACTS_DIR="gcs_artefacts"
+SCALABILITY_ARTIFACTS_LOCATION="$WORKSPACE/$GCS_ARTIFACTS_DIR"
+GCS_BUCKET="gs://kibana-performance/scalability-tests"
+
+# These tests are running on static workers so we must delete previous Kibana build and scalability artifacts
+rm -rf "$KIBANA_BUILD_LOCATION"
+rm -rf "$SCALABILITY_ARTIFACTS_LOCATION"
+
+echo "--- Download the latest artifacts from single user performance pipeline"
+source .buildkite/scripts/steps/scalability/download_artifacts.sh
+
+echo "--- Clone kibana-load-testing repo and prepare project"
+source .buildkite/scripts/steps/scalability/build_load_runner.sh
+
+echo "--- Run Scalability Tests with Elasticsearch started only once and Kibana restart before each journey"
+cd "$KIBANA_DIR"
+node scripts/es snapshot&
+
+esPid=$!
+
+# unset env vars defined in other parts of CI for automatic APM collection of
+# Kibana. We manage APM config in our FTR config and performance service, and
+# APM treats config in the ENV with a very high precedence.
+unset ELASTIC_APM_ENVIRONMENT
+unset ELASTIC_APM_TRANSACTION_SAMPLE_RATE
+unset ELASTIC_APM_SERVER_URL
+unset ELASTIC_APM_SECRET_TOKEN
+unset ELASTIC_APM_ACTIVE
+unset ELASTIC_APM_CONTEXT_PROPAGATION_ONLY
+unset ELASTIC_APM_GLOBAL_LABELS
+unset ELASTIC_APM_MAX_QUEUE_SIZE
+unset ELASTIC_APM_METRICS_INTERVAL
+unset ELASTIC_APM_CAPTURE_SPAN_STACK_TRACES
+unset ELASTIC_APM_BREAKDOWN_METRICS
+
+export TEST_ES_URL=http://elastic:changeme@localhost:9200
```

Comment:
We could just move up the definition of `ES_HOST` and use that one instead if we provide username and password via `curl -u elastic:changeme ...`.